### ISSUE TITLE
Warn of minor conflict with GitHub Sponsors

### ISSUE THREAD

**Issue by jgneff** [liberapay/liberapay.com#1709]
There is a minor conflict when signing up for both GitHub Sponsors and Liberapay: you need to use two separate e-mail addresses when registering the services with Stripe. I find it annoying when the only way around a problem is to “get another e-mail address,” but that was the solution offered by the support representatives at both GitHub and Stripe.

The details are in my [GitHub Community Forum](https://github.community/t5/How-to-use-Git-and-GitHub/GitHub-Sponsors-Get-access-to-Stripe-account/m-p/50311) question with the title “GitHub Sponsors: Get access to Stripe account?”

The summary I posted there states, “Yes, it seems that using two different e-mail addresses is the only way to enroll in GitHub Sponsors while also having access to a full Stripe account for use in other contexts.” Liberapay is the *other contexts* to which I was referring. Does anyone here know of a way to connect both Liberapay and GitHub Sponsors into the same Stripe account?

By the way, I'm impressed with the Liberapay website and interface—simple and slick! Thank you.

**Comment by Changaco** [comment]
I didn't know GitHub Sponsors used Stripe Express accounts. It seems to me that this problem can only really be fixed by Stripe, they should probably allow you to “upgrade” your Express account to a full one, or allow you to create a full account and attach the Express account to it.

The more platforms adopt Express accounts, the more people will find themselves in the same confusing situation.

**Comment by jgneff** [comment]
> It seems to me that this problem can only really be fixed by Stripe, they should probably allow you to “upgrade” your Express account to a full one, or allow you to create a full account and attach the Express account to it.

Right! Meanwhile, I think it would be nice to have a one-sentence warning on the Liberapay website about how this works. I assumed that GitHub Sponsors and Liberapay connected to Stripe in the same way, so I waited to finish setting up Liberapay so I could create the Stripe account through GitHub first. If I had known that GitHub creates a hidden Stripe account that requires a separate e-mail address, I would have gone live with Liberapay right away.

I'll see whether I can come up with a good pull request to add a brief statement on either the Payment Processors page or the FAQ page.

**Comment by jgneff** [comment]
I don't have the versions of the tools required to build and run the Liberapay website on my system, so to save some time, may I start by suggesting the changes here? Below is my _untested_ suggestion for changes.

I propose adding the following paragraph to the end of the Payment Processors page (_www/about/payment-processors.spt_), formatted below to minimize the horizontal scrolling here on GitHub:

```html
<p>{{ _(
    "Note that the full-featured Stripe account used by Liberapay must be
     registered with a {issue_1709_link_open}different e-mail address{link_close}
     than any Stripe Express accounts you create for other platforms, most
     notably the Stripe Express account created through GitHub Sponsors.",
    issue_1709_link_open='<a href="https://github.com/liberapay/liberapay.com/issues/1709">'|safe,
) }}</p>
```

**Comment by Changaco** [comment]
That's good. However, instead of simply adding that paragraph to the page, maybe we should create an intermediate page displayed after the user clicks on the “Connect a Stripe account” button, so that we can show the user more information, without overloading the existing page. This intermediate page could help fix #1186.

**Comment by trebmuh** [comment]
agreed

**Comment by jgneff** [comment]
That's fine with me. I do remember pushing some "Connect a Stripe account" button to investigate what was on the other side. I saw the Liberapay/Stripe page and decided to wait and set up GitHub Sponsors first (which took six weeks). A note up front, even on an intermediate page, might allow someone to go ahead and fill out the Liberapay/Stripe page with confidence instead of waiting.

Also, anyone is welcome to do the pull request for this issue. I don't have the time right now to set up the proper Liberapay build and test environment. The text I proposed is as follows, but you're welcome to make any changes to it.

> Note that the full-featured Stripe account used by Liberapay must be registered with a different e-mail address than any Stripe Express accounts you create for other platforms, most notably the Stripe Express account created through GitHub Sponsors.
